spec/folio_client_spec.rb

```diff
@@ -472,7 +472,6 @@
   context "when token is expired" do
     let(:inventory) { instance_double(FolioClient::Inventory, fetch_hrid: nil) }
     let(:hrid) { "in56789" }
-    let(:expired_token) { "expired_token" }
```

Comment:
Re-use the token from above. That's the one we'll consider expired, because it already is a bogus token which the API endpoint considers expired.